src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -124,5 +124,12 @@ Integer getInt(Map<Option, ?> options) {
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
-  // TODO(ajaykannan): implement "Organization" functionality when available (issue #319)
+  /**
+   * Tests whether the caller has the given permissions on the specified Organization. Returns a
+   * list of booleans corresponding to whether or not the user has the permission in the same
```

Comment:
Do we maybe just want the logical AND of all these permissions?

Or perhaps what you want to return is a Map<String, boolean> that lists the permissions and their results?